UnitsNet/GeneratedCode/Quantities/Angle.g.cs

```diff
@@ -224,6 +221,11 @@ public Angle(double value, AngleUnit unit)
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
         QuantityInfo<AngleUnit> IQuantity<AngleUnit>.QuantityInfo => Info;
 
+#if NETSTANDARD2_0
+        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
+        IQuantityInstanceInfo<Angle> IQuantityInstance<Angle>.QuantityInfo => Info;
+#endif
```

Comment:
Out of scope, but I don't find `IQuantityInstanceInfo` and `IQuantityInstance` names intuitive. 

It says
> This is a specialization of IQuantityInfo that is used (internally) for constraining certain methods, without having to include the unit type as additional generic parameter.

1. Should they be `internal`?
2. How about naming them `ITypedQuantityInfo` and `ITypedQuantity` or similar?

Reply:
1. Wish it could, but that would make it impossible to implement in the `HowMuch` (besides, it's being extended by another public interface).
2. I don't now, the thing I was trying to convey with the name (unsuccessfully)  was that when using it as a constraint (e.g. for a collection of quantities)- all quantities would be _instances_ of the the `T` parameter. What about `IQuantityOfType`?

Reply:
I like `IQuantityOfType<T>`